README.md

```diff
@@ -44,59 +50,53 @@ cd generative_data_prep
 pip install .
 ```
 
-## Requirements
-- Python version 3.9+, **only verified on python 3.9**
-- Support for linux and mac OS. Not tested on Windows
+</br>
```

Comment:
I added these breaks in between Header2 sections since the visual separation is a little easier on the eyes.